ddtrace/llmobs/_llmobs.py

```diff
@@ -571,11 +572,12 @@ def enable(
 
     @classmethod
     def pull_dataset(cls, name: str) -> Dataset:
-        return cls._instance._dne_client.dataset_pull(name)
+        ds = cls._instance._dne_client.dataset_get_with_records(name)
+        return ds
 
     @classmethod
-    def create_dataset(cls, name: str, description: str) -> Dataset:
-        return cls._instance._dne_client.dataset_create(name, description)
+    def create_dataset(cls, name: str, description: str, records: List[DatasetRecord] = []) -> Dataset:
+        return cls._instance._dne_client.dataset_create_with_records(name, description, records)
```

Comment:
@gary-huang if someone creates a brand new dataset with records, i.e. `create_dataset(name, records=[...])`, would we expect the dataset version to be 0 or 1? 

we're implementing it such that we create the dataset and then update it, so we should be expecting the dataset version from creating it to be 1 (not 0 if zero-indexing) right?

Reply:
looks like it's 1-indexed to me

Reply:
oops missed this, yea it's1 indexed